src/test/java/org/terasology/module/behaviors/MovementTests.java

```diff
@@ -182,6 +182,32 @@
                         0.9f,
                         0.3f,
                         new String[]{"walking", "leaping"}
+                ),
+                Arguments.of(
+                        "diagonally early up",
```

Comment:
what do you mean by "early" and "late" here?

Reply:
early in the sense that - assuming we don't actually walk diagonally but first to one side and then back into the diagonal - the first step brings us to the desired y-level and the second brings us on the same y-level to our target block
late vice versa means we first walk on the same y-level and then change y-levels to the desired one.

it's weird to explain and even weirder to name, so I'm happy about any naming help here